docs/resources.md

```diff
@@ -0,0 +1,208 @@
+default    ├─ConfigMap/cassandra-instance-cassandra-exporter-config-yml     -              6m9s
+default    ├─ConfigMap/cassandra-instance-generate-cassandra-yaml           -              6m9s
+default    ├─ConfigMap/cassandra-instance-generate-cqlshrc-sh               -              6m9s
+default    ├─ConfigMap/cassandra-instance-generate-nodetool-ssl-properties  -              6m9s
+default    ├─ConfigMap/cassandra-instance-generate-tls-artifacts-sh         -              6m9s
+default    ├─ConfigMap/cassandra-instance-jvm-options                       -              6m9s
+default    ├─ConfigMap/cassandra-instance-node-scripts                      -              6m9s
+default    ├─ConfigMap/cassandra-instance-topology-lock                     -              6m9s
+default    ├─PodDisruptionBudget/cassandra-instance-pdb                     -              6m9s
+default    ├─Role/cassandra-instance-node-role                              -              6m10s
+default    ├─Role/cassandra-instance-role                                   -              6m9s
+default    ├─RoleBinding/cassandra-instance-binding                         -              6m9s
+default    ├─RoleBinding/cassandra-instance-node-default-binding            -              6m10s
+default    ├─Secret/cassandra-instance-tls-store-credentials                -              6m9s
+default    ├─Service/cassandra-instance-svc                                 -              6m9s
+default    ├─ServiceAccount/cassandra-instance-sa                           -              6m9s
+default    ├─ServiceMonitor/cassandra-instance-monitor                      -              6m9s
+default    └─StatefulSet/cassandra-instance-node                            -              6m9s
+default      ├─ControllerRevision/cassandra-instance-node-659c89769d        -              2m24s
+default      ├─Pod/cassandra-instance-node-0                                True           2m14s
+default      ├─Pod/cassandra-instance-node-1                                True           94s
+default      └─Pod/cassandra-instance-node-2                                True           53s
+```
+
+### Statefulset
+
+Statefulsets are designed to manage stateful workload in Kubernetes. KUDO
+Cassandra uses statefulsets. The operator by default uses `OrderedReady`pod
+management policy. Which guarantees that pods are created sequentially. This
+makes sure that when the Cassandra cluster is coming up, only one node starts at
+the same time. Pod names are <instance-name>-node-<ordinal-index> starting from
+ordinal-index 0. For example a 3 node cluster created using KUDO Cassandra
+instance name cass-prod will have these pods:
+
+```
+$ kubectl get pods
+NAME               READY   STATUS    RESTARTS   AGE
+cass-prod-node-0   1/1     Running   0          101s
+cass-prod-node-1   1/1     Running   0          49s
+cass-prod-node-2   1/1     Running   0          8s
+```
+
```

Comment:
Note: When a multi-datacenter configuration with NODE_TOPOLOGY is used, the pod names include the datacenter name as well. See (link to Multi-DC-Setup documentation).

Reply:
added!